chill-algebird/src/test/scala/com/twitter/chill/algebird/AlgebirdSerializersSpec.scala

```diff
@@ -16,15 +16,22 @@ limitations under the License.
 
 package com.twitter.chill.algebird
 
-import com.twitter.chill.{ KSerializer, ScalaKryoInstantiator, KryoPool }
-import com.twitter.algebird.{ AveragedValue, DecayedValue, HyperLogLogMonoid, MomentsGroup, AdaptiveVector }
+import com.twitter.chill.{ ScalaKryoInstantiator, KryoPool }
+import com.twitter.algebird.Operators._
+import com.twitter.algebird._
+
 import org.scalatest._
 
+
 class AlgebirdSerializersSpec extends WordSpec with Matchers {
+  implicit def string2Bytes(i: String) = i.toCharArray.map(_.toByte)
+  implicit val skmMonoid = SketchMap.monoid[String, Long](SketchMapParams[String](1, 0.01, 1e-3, 10))
+
   val kryo = {
     val inst = () => {
       val newK = (new ScalaKryoInstantiator).newKryo
       newK.setReferences(false) // typical in production environment (scalding, spark)
+      newK.register(classOf[com.twitter.algebird.SketchMap[_, _]], new SketchMapSerializer)
       (new AlgebirdRegistrar).apply(newK)
```

Comment:
Yeah, I have tried it before and, sorry, I do not know how to do that generic way. The `SketchMapSerializer` requires the implicit `SketchMapMonoid`, which need to be specified before, but to do that I need to create one with very specific `SketchMapParams` as in 
` implicit val skmMonoid = SketchMap.monoid[String, Long](SketchMapParams[String](1, 0.01, 1e-3, 10))`. Can you take a look and help? Thank you.